setup.cfg

```
@@ -38,32 +38,17 @@ select =
 [mypy]
 ignore_missing_imports = True
 
-[mypy-cudf._lib.*]
```

I'm now `--exclude`ing these modules, rather than parsing and then ignoring errors.

---

Smart. Should we do the same for the _version files?

---

Good idea, done.